posts/2024-04-23-24.0.0.4.adoc

```diff
@@ -0,0 +1,168 @@
+
+For more information about Java 22, see the following resources:
+
+* link:https://jdk.java.net/22/release-notes[Release notes]
+* link:https://docs.oracle.com/en/java/javase/22/docs/api/index.html[Javadocs]
+* link:https://docs.oracle.com/en/java/javase/22/migrate/index.html[Migration guide]
+* link:https://adoptium.net/temurin/releases/?version=22[Download page]
+
+   
+// DO NOT MODIFY THIS LINE. </GHA-BLOG-TOPIC> 
+
+// // // // DO NOT MODIFY THIS COMMENT BLOCK <GHA-BLOG-TOPIC> // // // // 
+// Blog issue: https://github.com/OpenLiberty/open-liberty/issues/28084
+// Contact/Reviewer: gkwan-ibm
+// // // // // // // // 
+
+[#guides]
+== 8 guides are updated to use MicroProfile Reactive Messaging 3.0
+
+The following 8 guides have been updated to use the MicroProfile Reactive Messaging 3.0, MicroProfile 6.1 and Jakarta EE 10 specifications:
+
+- link:https://openliberty.io/guides/microprofile-reactive-messaging.html[Creating reactive Java microservices]
+- link:https://openliberty.io/guides/microprofile-reactive-messaging-acknowledgment.html[Acknowledging messages using MicroProfile Reactive Messaging]
+- link:https://openliberty.io/guides/microprofile-reactive-messaging-rest-integration.html[Integrating RESTful services with a reactive system]
+- link:https://openliberty.io/guides/microprofile-rest-client-async.html[Consuming RESTful services asynchronously with template interfaces]
+- link:https://openliberty.io/guides/reactive-messaging-sse.html[Streaming updates to a client using Server-Sent Events]
+- link:https://openliberty.io/guides/reactive-rest-client.html[Consuming RESTful services using the reactive JAX-RS client]
+- link:https://openliberty.io/guides/reactive-service-testing.html[Testing reactive Java microservices]
+- link:https://openliberty.io/guides/cloud-openshift-operator.html[Deploying microservices to OpenShift 4 using Kubernetes Operators]
+
+The integration tests in those guides have been updated to use Testcontainers. To learn how to test reactive Java microservices in true-to-production environments using Testcontainers, try out the link:https://openliberty.io/guides/reactive-service-testing.html[Testing reactive Java microservices] guide.
+   
+// DO NOT MODIFY THIS LINE. </GHA-BLOG-TOPIC> 
+
+
+[#CVEs]
+== Security vulnerability (CVE) fixes in this release
+[cols="6*"]
+|===
+|CVE |CVSS score by X-Force® |Vulnerability assessment |Versions affected |Version fixed |Notes
+
+|http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-51775[CVE-2023-51775]
+|7.5
+|Denial of service
+|21.0.0.3 - 24.0.0.3
+|24.0.0.4
+|Affects the feature:openidConnectClient-1.0[], feature:socialLogin-1.0[], feature:mpJwt-1.2[], feature:mpJwt-2.0[], feature:mpJwt-2.1[], feature:jwt-1.0[] features
```

Comment:
the feature macros wont work in blogs, so replace them w/ relative URLs, eg

link:{url-prefix}/docs/latest/reference/feature/openidConnectClient-1.0.html[openidConnectClient-1.0]